jekyll/_cci2/server-3-operator-nomad.adoc

```diff
@@ -0,0 +1,118 @@
+
+=== Checking the Jobs Status
+
+The get a list of statuses for all jobs in your cluster, run:
+
+```shell
+kubectl exec -it nomad-pod-IDm -- nomad status
+```
+
+The `Status` is the most important field in the output, with the following status type definitions:
+
+- `running`: Nomad has started executing the job. This typically means your job in CircleCI is started.
+
+- `pending`: There are not enough resources available to execute the job inside the cluster.
+
+- `dead`: Nomad has finished executing the job. The status becomes `dead` regardless of whether the corresponding CircleCI job/build succeeds or fails.
+
+=== Checking the Cluster Status
+
+To get a list of your Nomad clients, run:
+
+```shell
+kubectl exec -it nomad-pod-IDm -- nomad node-status
+```
+
+NOTE: `nomad node-status` reports both Nomad clients that are currently serving (status `active`) and Nomad clients that were taken out of the cluster (status `down`). Therefore, you need to count the number of `active` Nomad clients to know the current capacity of your cluster.
+
+To get more information about a specific client, run the following from that client:
+
+```shell
+kubectl exec -it nomad-pod-IDm -- nomad node-status -self
+```
+
+This will give information such as how many jobs are running on the client and the resource utilization of the client.
+
+=== Checking Logs
+
+As noted in the Nomad Jobs section above, a Nomad Job corresponds to an execution of a CircleCI job. Therefore, Nomad Job logs can sometimes help to understand the status of a CircleCI job if there is a problem. To get logs for a specific job, run:
+
+```shell
+kubectl exec -it nomad-pod-IDm -- nomad logs -job -stderr <nomad-job-id>
+```
+
+NOTE: Be sure to specify the `-stderr` flag as this is where most Build Agent logs appear.
+
+While the `nomad logs -job` command is useful, the command is not always accurate because the `-job` flag uses a random allocation of the specified job. The term `allocation` is a smaller unit in Nomad Job, which is out of scope for this document. To learn more, please see https://www.nomadproject.io/docs/internals/scheduling.html[the official document].
+
+Complete the following steps to get logs from the allocation of the specified job:
+
+. Get the job ID with `nomad status` command.
+. Get the allocation ID of the job with `nomad status <job-id>` command.
+. Get the logs from the allocation with `nomad logs -stderr <allocation-id>`
+
+=== Shutting Down a Nomad Client
+
+When you want to shutdown a Nomad client, you must first set the client to `drain` mode. In `drain` mode, the client will finish any jobs that have already been allocated but will not be allocated any new jobs.
+
+. To drain a client, log in to the client and set the client to drain mode with `node-drain` command as follows:
++
+```shell
+kubectl exec -it nomad-pod-IDm -- nomad node-drain -self -enable
```

Comment:
Ah this section is a bit different from the rest. This is done on the Nomad client where there is no kubectl - they can juts run the nomad commands directly.
```suggestion
nomad node-drain -self -enable
```

Reply:
Ah right I see, thank you!